docs/hpx.idx

```diff
@@ -102,6 +102,13 @@ parallel::generate_n                  "generate_n" "hpx\.parallel\.v1\.generate_
 # hpx/parallel/algorithms/inclusive_scan.hpp
 parallel::inclusive_scan              "inclusive_scan" "hpx\.parallel\.v1\.inclusive_scan.*"
 
+# hpx/parallel/algorithms/is_partitioned.hpp
+parallel::is_partitioned              "is_partitioned" "hpx\.parallel\.v1\.is_partitioned.*"
+
+# hpx/parallel/algorithms/is_sorted.hpp
+parallel::is_sorted                   "is_sorted"  "hpx\.parallel\.v1\.is_sorted.*"
```

Comment:
This regex will match `is_sorted_until` as well which will create a wrongly placed index entry.